### PR TITLE
SLVS-1371 Add listUserOrganizations to ConnectionRpcService

### PR DESCRIPTION
**Pull request by gabriela-trutan-sonarsource**

### CODE REVIEW

File: src/SLCore/Service/Connection/ListUserOrganizationsResponse.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Diagnostics.CodeAnalysis;
+using SonarLint.VisualStudio.SLCore.Service.Connection.Models;
+
+namespace SonarLint.VisualStudio.SLCore.Service.Connection;
+
+[ExcludeFromCodeCoverage]
```
Comment (vnaskos-sonar) [inline src/SLCore/Service/Connection/ListUserOrganizationsResponse.cs:26]: We should have a test for this one as well. I believe it's kinda a convention, to check if it searilizes correctly.

File: src/SLCore/Service/Connection/Models/OrganizationDto.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace SonarLint.VisualStudio.SLCore.Service.Connection.Models;
+
+[ExcludeFromCodeCoverage]
```
Comment (vnaskos-sonar) [inline src/SLCore/Service/Connection/Models/OrganizationDto.cs:25]: You can safely remove the exclusion once the `ListUserOrganizationsResponse` has a test

**Review by vnaskos-sonar — CHANGES_REQUESTED** [review]

File: src/SLCore/Service/Connection/ListUserOrganizationsResponse.cs (hunk 1 above)
Comment (gabriela-trutan-sonarsource) [reply src/SLCore/Service/Connection/ListUserOrganizationsResponse.cs:26]: I think we should discuss this point. It feels pointless to test the JsonConvert from Newtonsoft over and over again, especially for records with simple structure.

Comment (gabriela-trutan-sonarsource) [reply src/SLCore/Service/Connection/ListUserOrganizationsResponse.cs]: I added it. We might add a more generic test using reflection, which will make these kind of tests obsolete, but it should probably be a separate PR

**Review by vnaskos-sonar — APPROVED** [review]
👍

**Pull request merged**